### PR TITLE
[doc] Adds new changes for 4.13

### PR DESCRIPTION
**Pull request by alosadagrande**
New things to be added to 4.13

### CODE REVIEW

File: documentation/modules/ROOT/pages/crafting-deployments-iac.adoc [hunk 1]
```diff
@@ -250,6 +257,26 @@ spec:
 EOF
 -----
 
+It is strongly recommended to include crun manifests as part of the additional install-time manifests for 4.13+. So let's create proper machine configuration in the extra manifests folder:
+
+[.console-input]
+[source,bash,subs="attributes+,+macros"]
+-----
+cat <<EOF > ~/5g-deployment-lab/ztp-repository/site-configs/hub-1/sno2-extra-manifest/enable-crun-master.yaml
```
Comment (mvazquezc) [inline documentation/modules/ROOT/pages/crafting-deployments-iac.adoc:265]: Shouldn't we add the same for workers? I know this is an SNO, but if in the future the SNO gets expanded to SNO+Workers these wouldn't have crun, right?

File: documentation/modules/ROOT/pages/crafting-cluster-telco-related-infra-operators-configs.adoc [hunk 2]
```diff
@@ -44,11 +44,14 @@ spec:
       metadata:
         name: redhat-operator-index
       spec:
-        image: infra.5g-deployment.lab:8443/redhat/redhat-operator-index:v4.12-1677697786 #TODO
+        image: infra.5g-deployment.lab:8443/redhat/redhat-operator-index:v4.13 #TODO
```
Comment (mvazquezc) [inline documentation/modules/ROOT/pages/crafting-cluster-telco-related-infra-operators-configs.adoc]: You could use this tag: v4.13-1688429207

built on: GMT: Tuesday, July 4, 2023 12:06:47 AM

File: lab-materials/lab-env-data/hub-cluster/hub.yml [hunk 3]
```diff
@@ -23,23 +23,25 @@ manifests_dir: hubmanifests
 disconnected_operators:
```
Comment (mvazquezc) [inline lab-materials/lab-env-data/hub-cluster/hub.yml:23]: Here need to change catalog link to

registry.redhat.io/redhat/redhat-operator-index:v4.13-1688429207

built on: GMT: Tuesday, July 4, 2023 12:06:47 AM

File: lab-materials/lab-env-data/hub-cluster/hub.yml [hunk 4]
```diff
@@ -23,23 +23,25 @@ manifests_dir: hubmanifests
...
 - name: openshift-gitops-operator
   channels:
   - name: latest
+  channels:
+  - name: latest
 - name: lvms-operator
   channels:
   - name: stable-4.13
 disconnected_extra_images:
 - registry.redhat.io/openshift4/ztp-site-generate-rhel8:v4.13
```
Comment (mvazquezc) [inline lab-materials/lab-env-data/hub-cluster/hub.yml]: use a specific tag? like v4.13.1-3

**Review by mvazquezc — COMMENTED** [review]
Some changes suggested.

File: lab-materials/lab-env-data/hub-cluster/hub.yml (hunk 4 above)
Comment (alosadagrande) [reply lab-materials/lab-env-data/hub-cluster/hub.yml]: fixed. Also included specific channels for gitops (gitops-1.9) and TALM (4.13)

File: lab-materials/lab-env-data/hub-cluster/hub.yml (hunk 3 above)
Comment (alosadagrande) [reply lab-materials/lab-env-data/hub-cluster/hub.yml:23]: Great, thanks!

File: documentation/modules/ROOT/pages/crafting-cluster-telco-related-infra-operators-configs.adoc (hunk 2 above)
Comment (alosadagrande) [reply documentation/modules/ROOT/pages/crafting-cluster-telco-related-infra-operators-configs.adoc]: Great, thanks!

File: documentation/modules/ROOT/pages/crafting-deployments-iac.adoc (hunk 1 above)
Comment (alosadagrande) [reply documentation/modules/ROOT/pages/crafting-deployments-iac.adoc:265]: Included worker machineconfig for crun too.

**Review by mvazquezc — APPROVED** [review]

**Pull request merged**